### PR TITLE
dotnet-sdk DEB and RPM package consume store by "dependency" field

### PR DESCRIPTION
**Pull request by wli3**
**Customer scenario**

Customer installs SDK via debian, RPM family package management. The method of getting runtime package store is changed from being part of dotnet-sdk package to obtain via native package manager which will download and install the runtime package store.

**Bugs this fixes**

https://github.com/dotnet/cli/issues/7154

**Workarounds, if any**

Install via install script

**Risk**

Low, there is no customer facing behavior change

**Performance impact**

During install time

**Root cause analysis**

N/A

**How was the bug found?**

N/A

@srivatsn for approval

### CODE REVIEW

**Comment by wli3** [comment]
@JunTaoLuo @joeloff FYI

**Comment by srivatsn** [comment]
Can you update the bug that this PR fixes? - it's pointing to this PR itself in the template above.

**Comment by wli3** [comment]
@srivatsn updated, this is a conclusion from a meeting with runtime and aspnet, I "backfilled" the issue

File: build/DependencyVersions.props [hunk 1]
```diff
@@ -35,10 +35,15 @@
...
-    <AspNetCoherenceLabel>rtm</AspNetCoherenceLabel>
+    <!--BranchName and AspNetCoreVersion will not always be the same-->
+    <AspNetCoreBranchName>2.0.0</AspNetCoreBranchName>
+    <AspNetCoreRelease>rtm</AspNetCoreRelease>
+    <AspNetCoreVersion>2.0.0</AspNetCoreVersion>
     <AspNetCoreCoherenceTimestamp>26239</AspNetCoreCoherenceTimestamp>
 
+    <AspNetCoreRuntimePackageBrandName>aspnetcore-store</AspNetCoreRuntimePackageBrandName>
+    <AspNetCoreVersionAndRelease>$(AspNetCoreVersion)-$(AspNetCoreRelease)</AspNetCoreVersionAndRelease>
+    <AspNetCoreRuntimeAzureblobStoresSubfolderName>$(AspNetCoreBranchName)-$(AspNetCoreCoherenceTimestamp)</AspNetCoreRuntimeAzureblobStoresSubfolderName>
```
Comment (johnbeisner) [inline build/DependencyVersions.props]: I think the "Coherence" part of "AspNetCoreCoherenceTimestamp" is misleading; maybe just "AspNetCoreTimestamp"

**Pull request merged**